src/utility/axios-request.jsx

```diff
@@ -44,3 +39,19 @@ export const postComment = (id, user, text) => {
       return response.data.comment;
     });
 };
+export const getReviewsWithCategory = (category) => {
+  return category === "reviews"
+    ? api.get("reviews").then((response) => {
+        return response.data.reviews;
+      })
+    : api
```

Comment:
You shouldn't need conditional logic here - the beauty of axios is that if it sees a param value is undefined it won't use that parameter, you can just put the params in